### PR TITLE
get location on click

### PR DESCRIPTION
**Pull request by LuanSSouza**
get location (latitude and longitude) on click in map

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -154,6 +154,16 @@
             layers: [satelliteLayer]
         });
 
+        var maker = null;
```
Comment (cardoso) [inline index.html]: marker

Comment (LuanSSouza) [reply index.html]: vlw

**Comment by vaniaon** [comment]
@LuanSSouza is there any issue associate with this PR?

**Comment by LuanSSouza** [comment]
Not yet. I'm creating

**Comment by LuanSSouza** [comment]
@vaniaon the issue associated with this PR is Get location by click on map #59

**Comment by cacarrara** [comment]
Closing it since #71 already fix the issue.

**Pull request closed without merging**